### PR TITLE
Upgrade notes updated

### PR DESCRIPTION
**Pull request by DerDanilo**
Cleaned up the order and put some eyecandy in. More to follow

**Info:** Cloned the wrong branch with edited onlyoffice file. Please merge the onlyoffice file before merging this one here.

### CODE REVIEW

File: deploy/only_office.md [hunk 1]
```diff
@@ -85,24 +85,38 @@ https://github.com/ONLYOFFICE/Docker-DocumentServer#recommended-system-requireme
...
-docker run -i -t -d -p 88:80 onlyoffice/documentserver --restart=always --name oods
+docker run -i -t -d -p 88:80 --restart=always --name oods onlyoffice/documentserver
 ```
 
+Ubuntu 16.04
+```
+docker run -dit -p 88:80 --restart always --name oods onlyoffice/documentserver
+```
+
+*Nothing yet confirmed on CentOS 7, you may try any of the above commands, they may work also.*
```
Comment (shoeper) [inline deploy/only_office.md]: The same change has been proposed in another PR as well, hasn't it?

File: upgrade/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Upgrade notes intro
+
+These notes give an overview of changes that have to be done when upgrading to specific version.
```
Comment (shoeper) [inline upgrade/README.md]: [...] to [a] specific [...]

**Comment by shoeper** [comment]
Personally I'd wrap all of the upgrade documents into one, with the instructions for the newest version at the top.

File: deploy/only_office.md (hunk 1 above)
Comment (DerDanilo) [inline deploy/only_office.md]: I messed something up with the branches, my fault. Please merge the other one first and then this one won't be an issue.

**Comment by DerDanilo** [comment]
We can do that later, I think that it's better to keep it seperate as this will allow us to only work on smaller parts of the documentation and also maybe merge it with the version specific information that is currently stored and shown somewhere else.

**Comment by shoeper** [comment]
I'd still merge them in one document.

**Comment by DerDanilo** [comment]
@freeplant @shoeper Did all that and cleaned up, please merge now, so I can tidy up the branches.
I also suggest that we tidy up any branches in the public repo if not required anymore.

**Comment by DerDanilo** [comment]
@freeplant When can we expect the merge?

**Comment by freeplant** [comment]
Sorry, I missed to notice that the PR was updated.

**Comment by DerDanilo** [comment]
Thanks!

**Pull request merged**